Apps/Sandcastle/gallery/CZML Points.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
Do you think we need this example given the `CZML Different Coordinate Maps` example?  Perhaps just rename that one to `CZML Points`?